ui/lib/config-ui/addon/components/messages/page/details.hbs

```diff
@@ -43,6 +43,16 @@
         (date-format (get @message attr.name) "MMM d, yyyy hh:mm aaa" withTimeZone=true)
       }}
     />
+  {{else if (eq attr.name "link")}}
+    {{#if (is-empty-value @message.link)}}
+      <InfoTableRow @label="Link" @value="None" />
+    {{else}}
+      {{#each-in @message.link as |title href|}}
+        <InfoTableRow @label="Link" @value={{title}}>
+          <Hds::Link::Inline @icon="external-link" @href={{href}} data-test-link="message link">{{title}}</Hds::Link::Inline>
```

Comment:
same here about `isHrefExternal`
```suggestion
          <Hds::Link::Inline @icon="external-link" @href={{href}} @isHrefExternal={{true}}>{{title}}</Hds::Link::Inline>
```

could you use the built in InfoTableRow selector combined with the `a` tag to assert this value instead of adding another select?